### ISSUE TITLE
Suggestion message for invalid commands

### ISSUE THREAD

**Issue by 0xWise64** [DEMON1A/Discord-Recon#16]
Summary

Suggestion response for invalid commands. To direct the user to using ".help".

Code:

```python
@client.event
async def on_command_error(ctx, error):
    if isinstance(error, commands.CommandNotFound):
        await ctx.send('**Invalid command, please type `.help` to see the list of commands and tools.**')
```

Regards

**Comment by DEMON1A** [comment]
Gonna test it on the production version then add it to the GitHub code, thanks for your help

**Comment by DEMON1A** [comment]
Closed by mistake, Sorry

**Comment by DEMON1A** [comment]
- This issue has been fixed at: 776a963
- Thanks for your help.
